src/string-class.js

```diff
@@ -0,0 +1,234 @@
+const stringClass = {
+  /**
+   * Has Vowels
+   * 
+   * It checks if a string contains any vowel letter.
+   * 
+   * @return {Boolean} True if string contains vowel and false if not.
+   */
+  hasVowels() {
+    const vowels = /[aeiou]/i;
+    return vowels.test(this);
+  },
+
+  /**
+   * To Upper
+   * 
+   * It converts string to uppercase.
+   * 
+   * @return {String} Uppercase of string.
+   */
+  toUpper() {
+    const lowerCase = /[a-z]/g;
+    return this.replace(lowerCase, (char) => {
+      return String.fromCharCode(char.charCodeAt() ^ 32);
+    });
+  },
+
+  /**
+   * To Lower
+   * 
+   * It converts string to lowercase.
+   * 
+   * @return {String} Lowercase of string.
+   */
+  toLower() {
+    const upperCase = /[A-Z]/g;
+    return this.replace(upperCase, (char) => {
+      return String.fromCharCode(char.charCodeAt() ^ 32);
+    });
+  },
+
+  /**
+   * Upper Character First
+   * 
+   * It converts the first character in a string to uppercase.
+   * 
+   * @return {String} First Character of string in uppercase.
+   */
+  ucFirst() {
+    return this.replace(this[0], this[0].toUpper());
+  },
+
+  /**
+   * Is Question
+   * 
+   * It checks if string is a question.
+   * 
+   * @return {Boolean} True if string is a question and false if not.
+   */
+  isQuestion() {
+    const quest = /^[\w\s]+\?$/g;
+    return quest.test(this.trim());
+  },
+
+  /**
+   * Words
+   * 
+   * It return the words of a string in an array.
+   * 
+   * @return {Array} Array of words in string.
+   */
+  words() {
+    const invalid = /[^\w\s]/g;
+    return this.replace(invalid, ' ').split(' ')
+    .filter((item) => {
+      return Boolean(item);
+    });
+  },
+
+  /**
+   * Word Count
+   * 
+   * It return the number words in a string.
+   * 
+   * @return {Number} The number of words in the string.
+   */
+  wordCount() {
+    return this.words().length;
+  },
+
+  /**
+   * Reverse Word
+   * 
+   * Reverse string.
+   * 
+   * @return {String} The reverse of the string.
+   */
+  reverseWord() {
+    return this.split('').reverse().join('');
+  },
+
+  /**
+   * To Currency
+   * 
+   * It changes number string to currency format.
+   * 
+   * @return {String} Currenct format of string.
+   */
+  toCurrency() {
+    const onThreeCount = /\d{3}/g;
```

Comment:
**Nitpick**
Initializing variables you don't use elsewhere might have some overhead. Not sure tho' but you can look it up